src/offline/UpdateHandlers/handleComments.ts

```diff
@@ -0,0 +1,55 @@
+import { OfflineContentRepository } from '../OfflineContentRepository';
+import { OfflineUpdateRequest } from '../OfflineUpdateRequest';
+import { OfflineUpdateRepository } from '../OfflineUpdateRepository';
+import { generateRandomId } from './utils';
+import { EntityType } from '../../typings/enums';
+import { APIComment } from '@equinor/procosys-webapp-components/dist/typings/apiTypes';
+
+const offlineContentRepository = new OfflineContentRepository();
+const offlineUpdateRepository = new OfflineUpdateRepository();
+
+export const handlePostComment = async (
+    offlinePostRequest: OfflineUpdateRequest
+): Promise<void> => {
+    const { Text: comment, PunchItemId: punchIdStr } =
+        offlinePostRequest.bodyData; // PunchId is stored in the body of the request
+    const newCommentId = generateRandomId();
+    const punchId = Number(punchIdStr);
+
+    if (punchId === null) {
+        console.error('The request parameters does not contain a punchId.');
+        return;
+        //todo: Hva gjør vi her?
+    }
+
+    const commentListEntity =
+        await offlineContentRepository.getEntityByTypeAndId(
+            EntityType.PunchComment,
+            punchId
+        );
+
+    const newComment = {
+        id: newCommentId,
+        text: comment,
+        firstName: '',
+        lastName: '',
+        createdAt: new Date(Date.now()).toLocaleString().split(',')[0],
```

Comment:
I just get invalid date here while offline. Is this done the same way as signing a checklist?